draftwatermark.dtx

```diff
@@ -808,6 +811,7 @@
 \DeclareStringOption[c]{hanchor}
 \DeclareStringOption[m]{vanchor}
 \DeclareStringOption[c]{alignment}
+\DeclareStringOption[0]{transparency} % 0 = invisible ; 1 = full opaque
```

Comment:
Shouldn't this be
```suggestion
\DeclareStringOption[1]{transparency} % 0 = invisible ; 1 = full opaque
```
to avoid making all watermarks fully invisible by default?